src/devices/Am2320/samples/Program.cs

```diff
@@ -22,8 +23,8 @@
 
 while (true)
 {
-    var temp = am2330.Temperature;
-    var hum = am2330.Humidity;
+    am2330.TryReadTemperature(out Temperature temp);
+    am2330.TryReadHumidity(out RelativeHumidity hum);
     if (am2330.IsLastReadSuccessful)
```

Comment:
I suggest to show how to do it right: `if (am2330.TryReadTemperature(out Temperature temp) && ...`

Reply:
Great catch! You're fully right!